tests/rt.sh

```diff
@@ -307,9 +307,10 @@ else
   die "Unknown machine ID, please edit detect_machine.sh file"
 fi
 
-mkdir -p ${STMP}/${USER}
+# mkdir -p ${STMP}/${USER}
```

Comment:
These changes will need to be reverted. and a new regression test will be required - maybe also remove an existing test, depending on how you handle the phasing out of cires_ugwp_v0.